sdk/keyvault/azure-security-keyvault-certificates/src/main/java/com/azure/security/keyvault/certificates/models/CertificateIssuer.java

```diff
@@ -85,6 +85,15 @@ public CertificateIssuer(String name, String provider) {
         this.provider = provider;
     }
 
+    /**
+     * Creates an instance of the issuer.
+     *
+     * @param name The name of the issuer.
+     */
+    public CertificateIssuer(String name) {
```

Comment:
Do we need this ?
Can create confusion for user as name and provider to create a valid issuer on the service side.

Reply:
But they are not required when you want to update so this 'ctor overload. Also, will mention in the methods where the provider should be required to clarify this.